src/login/logind-dbus.c

```diff
@@ -1191,7 +1191,7 @@ static int method_release_session(sd_bus_message *message, void *userdata, sd_bu
                 return r;
 
         if (session != sender_session)
-                return sd_bus_error_set(error, BUS_ERROR_NOT_IN_CONTROL, "You are not in control of this session");
+                return sd_bus_error_set(error, SD_BUS_ERROR_ACCESS_DENIED, "You are not in control of this session");
```

Comment:
please also change the text, i.e. maybe say "Refused to release session, since it is not yours" or so?

i.e. i think the wording "in control of this session" should be used only for doing session controller related operations, which this doesn't do

Reply:
or maybe "Refused to release session, since it doesn't match the one of the client"?